### DISCUSSION TITLE
Very slow adding where clause

### DISCUSSION THREAD

**Discussion by DineshS-nd** [sfu-db/connector-x#344] [Q&A]
It is very fast when I run ""select * from table limit 1000000". But very slow when adding where clause.
"select time_stamp, driver_id, vehicle_id  from table  where time_stamp >= '2022-08-10' and time_stamp < '2022-08-11' 
and driver_id>0 and  vehicle_id>0  limit 10000" .
Please explain the reason behind and how to overcome this problem.

**Comment by wangxiaoying** [comment]
Hi @DineshS-nd , can you test the execution time of the two queries: "select * from table limit 1000000" and "select time_stamp, driver_id, vehicle_id from table where time_stamp >= '2022-08-10' and time_stamp < '2022-08-11'
and driver_id>0 and vehicle_id>0 limit 10000" on the database server? I suspect that the difference is caused by server execution (e.g. scanning time).
